
# HG changeset patch
# User prr
# Date 1521484190 25200
# Node ID 3e3696a308e1674a1056d5ff0d660e7fb0d08286
# Parent  79f6a4dc221e6290723e773813969de422733545
8198649: Switch AWT/Swing's default GTK version to 3
Reviewed-by: psadhukhan, kaddepalli

diff -r 79f6a4dc221e -r 3e3696a308e1 src/java.desktop/unix/native/libawt_xawt/awt/gtk_interface.c
--- a/src/java.desktop/unix/native/libawt_xawt/awt/gtk_interface.c	Mon Mar 19 10:46:31 2018 -0700
+++ b/src/java.desktop/unix/native/libawt_xawt/awt/gtk_interface.c	Mon Mar 19 11:29:50 2018 -0700
@@ -45,18 +45,18 @@
 
 static GtkLib gtk_libs[] = {
     {
+        GTK_3,
+        JNI_LIB_NAME("gtk-3"),
+        VERSIONED_JNI_LIB_NAME("gtk-3", "0"),
+        &gtk3_load,
+        &gtk3_check
+    },
+    {
         GTK_2,
         JNI_LIB_NAME("gtk-x11-2.0"),
         VERSIONED_JNI_LIB_NAME("gtk-x11-2.0", "0"),
         &gtk2_load,
         &gtk2_check
-    },
-    {
-        GTK_3,
-        JNI_LIB_NAME("gtk-3"),
-        VERSIONED_JNI_LIB_NAME("gtk-3", "0"),
-        &gtk3_load,
-        &gtk3_check
     }
 };
 

